### PR TITLE
Cm azure db

### PR DESCRIPTION
**Pull request by xtreampb**
### Step template checklist

- [ ] `Id` should be a **GUID** that is not `00000000-0000-0000-0000-000000000000`
  - **NOTE** If you are modifying an existing step template, please make sure that you **do not** modify the `Id` property *(updating the `Id` will break the Library sync functionality in Octopus)*. 
- [ ] `Version` should be incremented, otherwise the integration with Octopus won't update the step template correctly
- [ ] Parameter names should not start with `$`
- [ ] **Step template parameter names (the ones declared in the JSON, not the script body) should be prefixed with a namespace so that they are less likely to clash with other user-defined variables in Octopus** (see [this issue](https://github.com/OctopusDeploy/Issues/issues/2126)). For example, use an abbreviated name of the step template or the category of the step template).
- [ ] `LastModifiedBy` field must be present, and (_optionally_) updated with the correct author
- [ ] If a new `Category` has been created:
   - [ ] An image with the name `{categoryname}.png` must be present under the `step-templates/logos` folder
   - [ ] The `switch` in the `humanize` function in [`gulpfile.babel.js`](https://github.com/OctopusDeploy/Library/blob/master/gulpfile.babel.js#L92) must have a `case` statement corresponding to it

### CODE REVIEW

File: step-templates/azure-database-import-create-new-vcpu.json [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+      "Octopus.Action.Azure.AccountId": "azureserviceprincipal-chris-azure-account-spaces-1"
```
Comment (MJRichardson) [inline step-templates/azure-database-import-create-new-vcpu.json:13]: This should be bound to a parameter :)

File: step-templates/azure-database-export.json [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Id": "85686923-5e57-4b1d-8dd7-6b5ca263ba42",
+      "Name": "azDbExport.backupName",
+      "Label": "Backup Name",
+      "HelpText": "The name of the file being saved.  Defaults to DB Name",
```
Comment (MJRichardson) [inline step-templates/azure-database-export.json:109]: Can we add to the help text to explain that .bacpac will be appended to the filename.

File: step-templates/azure-database-import-create-new-vcpu.json (hunk 1 above)
Comment (xtreampb) [reply step-templates/azure-database-import-create-new-vcpu.json]: Corrected in change f0a0ded

File: step-templates/azure-database-export.json (hunk 2 above)
Comment (xtreampb) [reply step-templates/azure-database-export.json]: corrected in a639433

**Pull request merged**